tests/e2e/api.opts

```diff
@@ -0,0 +1,7 @@
+--timeout 2200000
```

Comment:
Was this file added there by mistake? I can not see CheckApi.spec.js in this PR.

Reply:
It is unnecessary file - has been removed